### DISCUSSION TITLE
iOS app does not display HDR photos

### DISCUSSION THREAD

**Discussion by danbee** [ente-io/ente#10978] [Enhancements]
### Description

The Mac desktop app correctly displays HDR photos, but in the iOS app the same photos just look dark and are clearly not being displayed in HDR.

These are AVIF format photos exported from Lightroom. iOS Photos shows them in full HDR.

Related: https://www.reddit.com/r/enteio/comments/1fepy46/photos_not_viewable_in_hdr_on_ente_ios_app/

### Version

0.9.71

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - iOS

**Comment by bverkron** [comment]
I really can't stress how much I want HDR in photos sharing apps like Ente.

I've been working on / off as a photographer for a couple decades and HDR is one of the biggest improvements to photo viewing I've ever experienced. As such I've moved most of my workflow to be HDR-first and am really struggling to find a platform that's capable of sharing/displaying HDR images properly.

Currently I'm running Ente Photo self-hosted for testing and while the web app does technically work for HDR it's got a lot of shortcomings compared to using a native smartphone app and makes the experience clunky for smartphone users (which let's face it is most of the end users device of choice these days).

**Comment by Midar** [comment]
The lack of HDR is quite sad. It's the main reason not to view photos on Ente on my phone ­– if it weren't for this, I'd have completely replaced the native photos app with Ente. But photos taken on my phone look just so incredibly dull on Ente compared to the native apps. And it doesn't matter if it's in JPEG or HEIC – it's dull. The desktop app at least shows HDR on the JPEG (but not the HEIC).

iOS supports HDR everywhere and it seems to be quite trivial to support: https://developer.apple.com/documentation/uikit/supporting-hdr-images-in-your-app
Can this be added in the next release, please?
